### PR TITLE
update contribution page and remove dashboard link

### PR DESCRIPTION
**Pull request by ayatnkw**

### CODE REVIEW

File: docs/help-community/contributing.md [hunk 1]
```diff
@@ -52,53 +52,20 @@ You can contribute to the n8n documentation, for example by documenting nodes or
...
-
-After the review, you'll have a chance to review any suggested changes and implement them. You can then notify the reviewer that your final draft is ready.
-
-#### 5. Publishing
-
-n8n will let you know an estimated publication date. 
-
-When the article is published, n8n will also promote it through social media. You're encouraged to share it on your own channels.
+1. Email your idea to [marketing@n8n.io](mailto:marketing@n8n.io) with the subject "Blog contribution: [Your Topic]."
+2. Submit your draft:
```
Comment (StarfallProjects) [inline docs/help-community/contributing.md:64]: There's no need to quote format this list. Just make it a normal list.

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
Remove quote formatting from list

File: docs/help-community/contributing.md (hunk 1 above)
Comment (ayatnkw) [reply docs/help-community/contributing.md:64]: Done :)

File: docs/help-community/contributing.md [hunk 2]
```diff
@@ -52,53 +52,20 @@ You can contribute to the n8n documentation, for example by documenting nodes or
...
+You can write an article for the [n8n blog](https://blog.n8n.io/){:target=_blank class=.external-link}. Your article can be, for example, a [workflow tutorial](https://blog.n8n.io/tag/tutorial/){:target=_blank class=.external-link}, an opinion piece on automation, or some domain-specific [automation guides](https://blog.n8n.io/tag/guide/){:target=_blank class=.external-link}.
 
-### Process to submit a post
+### How to submit a post
 
-n8n appreciates all contributions. Publishing a tutorial on your own site that supports the community is a great contribution. However, if you want n8n to highlight your post on our blog, you need to go through this process.
+n8n appreciates all contributions. Publishing a tutorial on your own site that supports the community is a great contribution. If you want n8n to highlight your post on the blog, you need to go through the following steps:
```
Comment (freakwriter) [inline docs/help-community/contributing.md:61]:
```suggestion
n8n appreciates all contributions. Publishing a tutorial on your own site that supports the community is a great contribution. If you want n8n to highlight your post on the blog, follow these steps: 
```

Comment (ayatnkw) [reply docs/help-community/contributing.md]: this looks great, I've committed your change — could you review & merge? @freakwriter

**Review by freakwriter — APPROVED** [review]

**Pull request merged**